### DISCUSSION TITLE
Base-stats information confused

### DISCUSSION THREAD

**Discussion by leohero92** [Lailloken/Exile-UI#481] [Q&A] [answered]
https://github.com/Lailloken/Lailloken-UI/wiki/Item-info#base-stats-information
I understand all stats except this 
![image](https://github.com/user-attachments/assets/81911109-0e2e-467e-9326-f2ddf64747e3)
This is base stats info on a Royal Plate - best STR based body armor, so as I understand 19% is not % compared to the best possible value within the item-class.

So what is this stat? And how is it calculated?

**Comment by Lailloken** [comment, marked as answer]
It's the base percentile of a defensive item, basically how close the defense stats are to the max roll of the given base-type.
![image](https://github.com/user-attachments/assets/84fb3f78-3d05-4c37-850b-3e6f1fcd4000)
(the tooltip would show 33% for this item)

If you assume this notation `armor: x (y - z)`, then the base percentile is calculated this way `(x-y) / (z-y)`. So `(561-530) / (625-530) = 0.326`, which is 33% when rounded up.
***
The trade site also displays this value:
![image](https://github.com/user-attachments/assets/f607081f-5df9-4c4f-9d1b-ac6ca0d92f76)

I'll add a clarification to the wiki.

**Reply by leohero92** [reply]
Thank you so much for detailed answer.
